conda_forge_webservices/update_me.py

```diff
@@ -23,14 +23,12 @@ def _run_git_command(args):
 
 
 def get_current_versions():
-    r = subprocess.run(["conda", "list"], capture_output=True)
-    out = r.stdout.decode("utf-8")
+    r = subprocess.run(["conda", "list", "--json"], capture_output=True, check=True)
+    out = json.loads(r.stdout.decode("utf-8"))
```

Comment:
Also in trying things locally, it looks like this saves us the `decode` step

```suggestion
    r = subprocess.run(["conda", "list", "--json"], capture_output=True, check=True, encoding="utf-8")
    out = json.loads(r.stdout)
```

Reply:
Another option might be

```suggestion
    r = subprocess.run(["conda", "list", "--json"], capture_output=True, check=True, text=True)
    out = json.loads(r.stdout)
```